packages/helper-backend/src/main.ts

```diff
@@ -2,17 +2,45 @@ import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
 import { ValidationPipe } from '@nestjs/common';
 import { useContainer } from 'class-validator';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
   app.enableCors();
 
   useContainer(app.select(AppModule), { fallbackOnErrors: true });
 
+  // Configure Swagger
+  const config = new DocumentBuilder()
+    .setTitle('LiteLLM-Langflow Proxy API')
+    .setDescription('API documentation for managing mappings and proxies between Langflow and LiteLLM.')
+    .setVersion('1.0')
+    .addTag('Deepchat Proxy', 'Endpoints for managing LLM provider proxies')
+    .build();
+
+  const document = SwaggerModule.createDocument(app, config, {
+    deepScanRoutes: true,
+    extraModels: []
+  });
+
+  SwaggerModule.setup('api', app, document, {
+    swaggerOptions: {
+      docExpansion: 'list',
+      defaultModelsExpandDepth: 3,
+      defaultModelExpandDepth: 3,
+      persistAuthorization: true
+    }
+  });
+
   // Apply validation pipe globally
   app.useGlobalPipes(
     new ValidationPipe({
-      transform: true
+      transform: true,
+      whitelist: true,
```

Comment:
I think this setting and the next setting are at odds with each other. I think we want to keep `whitelist`